.github/workflows/tests.yml

```diff
@@ -1,16 +1,19 @@
 name: Tests
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Also sneaked that change in so we would have caught that issue earlier:
https://github.com/amitdev/lru-dict/pull/49#issuecomment-1666815977